Calendar/Parse Handlers/AuthenticationHandler.m

```diff
@@ -0,0 +1,62 @@
+//
+//  AuthenticationHandler.m
+//  InstagramClone
+//
+//  Created by Angelina Zhang on 6/27/22.
+//
+
+#import "AuthenticationHandler.h"
+#import <Parse/Parse.h>
+
+@implementation AuthenticationHandler
+
+- (BOOL)fieldsAreEmpty:(NSString *)username
+          withPassword:(NSString *)password {
```

Comment:
Nit: Usually convention is the first argument gets 'with' and the second one is just there. `fieldsAreEmptyForUsername:(NSString *)username password:(NSString *)password`